### ISSUE TITLE
Add Auto scroll on Account page

### ISSUE THREAD

**Issue by manoharreddyporeddy** [aropan/clist#283]
@aropan Thanks for the work you put in the clist website

===

Link 1
50 records are shown, and there is auto scroll when we go down in below page:
https://clist.by/accounts/?resource=102&sort_column=rating&sort_order=desc&country=IN

Link 2
10 records are shown, and there is  no  auto scroll when we go down in below page, we have to press 'more' button:
https://clist.by/account/pgmreddy@.com/resource/leetcode.com/

In the 2nd link above, we have to press many times to see all contests
Please add Auto scroll on 2nd link page, similar to 1st link above.

**Comment by aropan** [comment]
Why you want to see all contests?

**Comment by manoharreddyporeddy** [comment]
I want to improve my performance in contests

So I solved all Q1 in all leetcode contests, now I am able to solve Q1 in 1 to 5 mins on average & always able to solve. I marked them all blue on clist now.

Later I started solving all Q2 in all contests, to be able to solve Q2 always. About 40 of them left now.

After a week or so, I will do same for Q3 & then Q4

Checking what I did not solve ( esp with some links not working & later enabled ), I am having to scroll multiple times.

==

If such an auto scroll is not optimal for all the users
then an option in preference page ( https://clist.by/settings/preferences/ ) will be ideal to have
For me such an option is critical for contests, also to raise bugs like this I generally scroll a lot on clist website.

===

@aropan
